ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -128,6 +135,9 @@ export const getRegistrationTasks = () => serverConfigStore.getValue() ? ([
       'and tools.',
     buttonText: 'Login',
     completedText: 'Linked',
+    completionTimestamp: (profile: Profile) => {
```

Comment:
i think we can do away with `isComplete` (at least the implementation) by just checking for `null` in `completionTimestamp`

I think leaving the function but changing it to call into `completionTimestamp` would also be good.

Also, we might be approaching the point where its worth formally defining this object as a class

Reply:
I got rid of it. Might make sense to be a class though, that would have made self reference easier.